numba/tests/test_array_methods.py

```diff
@@ -917,6 +921,18 @@ def check(a):
         check(np.array([[3.1, 3.1], [1.7, 2.29], [3.3, 1.7]]))
         check(np.array([]))
 
+    def test_repeat(self):
```

Comment:
I know there are other functions that are testing non-"array methods" in this file, however I think that this test should go in `numba/tests/test_np_functions.py` (sorting out the tests into better categories is an ongoing project!).